dot/rpc/modules/author_test.go

```diff
@@ -314,20 +272,39 @@ func TestAuthorModule_PendingExtrinsics(t *testing.T) {
 				coreAPI:    tt.fields.coreAPI,
 				txStateAPI: tt.fields.txStateAPI,
 			}
-			if err := cm.PendingExtrinsics(tt.args.r, tt.args.req, tt.args.res); (err != nil) != tt.wantErr {
-				t.Errorf("AuthorModule.PendingExtrinsics() error = %v, wantErr %v", err, tt.wantErr)
-				return
-			}
-			if diff := cmp.Diff(tt.wantRes, *tt.args.res); diff != "" {
-				t.Errorf("unexpected response: %s", diff)
+			res := PendingExtrinsicsResponse{}
+			err := cm.PendingExtrinsics(tt.args.r, tt.args.req, &res)
+			if tt.expErr != nil {
+				assert.EqualError(t, err, tt.expErr.Error())
+			} else {
+				assert.NoError(t, err)
```

Comment:
Just a FYI (don't do it here), I came up (like a week ago or so) with a newer way to assert errors, example:

https://github.com/ChainSafe/gossamer/blob/91ed71612dd7fc72370f212e97e5e4d7fba96743/lib/trie/leaf/header_test.go#L64-L69

That's nice to assert the wrapped error is the expected one as well as to check the error message.
As in, **eventually** use that in the future. Your method (which was mine for years as well :smile:) works quite fine too so no worry.